composer.json

```diff
@@ -37,14 +37,18 @@
     },
     "require": {
         "php": ">=5.5",
-        "guzzle/guzzle": "3.9.*",
         "firebase/php-jwt": "2.2.*",
         "phpseclib/phpseclib": "0.3.*|~1.0|~2.0",
         "cache/taggable-cache": "0.4.*",
         "cache/redis-adapter": "0.4.*",
         "cache/memcached-adapter": "0.3.*",
         "cache/array-adapter": "0.4.*",
-        "cache/void-adapter": "0.3.*"
+        "cache/void-adapter": "0.3.*",
+        "psr/http-message": "^1.0",
+        "php-http/httplug": "^1.0",
+        "php-http/discovery": "^1.0",
+        "php-http/guzzle6-adapter": "^1.1",
+        "guzzlehttp/psr7": "^1.3"
```

Comment:
You should try to avoid this. Depend on the `php-http/httplug` instead of the `php-http/guzzle6-adapter` and PSR7 implementaiton